### PR TITLE
OTEL HTTP receiver checks enhancement

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

7/24 update - followed up w/SMEs

[OTEL ping checks enhancement](https://app.slack.com/client/T024F4SFX/C01KD5GHQ5C/thread/C01KD5GHQ5C-1683551746.023889). 

Do not merge! Waiting on [SMEs to add process steps](https://sumologic.atlassian.net/browse/SUMO-221966).

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204564870906120

### CODE REVIEW

File: docs/send-data/opentelemetry-collector/faq.md [hunk 1]
```diff
@@ -1,16 +1,17 @@
 ---
 id: faq
 title: FAQ
-description: Frequently asked questions about Sumo Logic OpenTelemetry Collector from our customers and field teams.
+description: Frequently asked questions about the Sumo Logic OpenTelemetry Collector from our customers and field teams.
 ---
+
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/faq.md:6]:
```suggestion
description: Frequently asked questions about Sumo Logic OpenTelemetry Collector from our customers and field teams.
---
```

File: docs/send-data/opentelemetry-collector/faq.md [hunk 2]
```diff
@@ -186,7 +186,17 @@ The `sumologicexporter` sends data to Sumo in batches. If the batches are small,
...
 If you are using the **Upstream OpenTelemetry collector** (instead of the **Sumo Logic OpenTelemetry collector**) and have a custom `config.yaml` file to collect data, then you might see this error. Sumo Logic expects specific tags on the data, which the Sumo Logic Distribution for OpenTelemetry Collector creates by default.
 
+## Ping checks
 
+#### Does Sumo OpenTelemetry support ping checks?
+
+Yes, our OpenTelemetry collector supports ping checks so that you can monitor up/down status for public https URLs.
+
+How it works - the OpenTelemetry receiver produces metrics that can be ingested and then monitored to track non-200 responses, latency, and other errors.
+
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/faq.md:196]:
```suggestion
<!--
1. To come...
1. ...
```

File: docs/send-data/opentelemetry-collector/faq.md [hunk 3]
```diff
@@ -186,7 +186,20 @@ The `sumologicexporter` sends data to Sumo in batches. If the batches are small,
...
+#### Does Sumo OpenTelemetry support ping checks?
+
+Yes, our OpenTelemetry collector supports ping checks so that you can monitor up/down status for public https URLs.
+
+How it works - the OpenTelemetry receiver produces metrics that can be ingested and then monitored to track non-200 responses, latency, and other errors.
+<!--
+1. To come...
+1. ...
+-->
+You can get alerted in 3 minutes or less when a OpenTelemetry collector disappears and is not marked ephemeral (i.e., server or VM crashes).
```
Comment (jpipkin1) [inline docs/send-data/opentelemetry-collector/faq.md:200]:
```suggestion
You can get alerted in 3 minutes or less when a OpenTelemetry collector disappears and is not marked ephemeral (that is, server or VM crashes).
```

**Review by jpipkin1 — APPROVED** [review]
Looks good, @kimsauce. Approved.

File: docs/send-data/opentelemetry-collector/faq.md [hunk 4]
```diff
@@ -186,7 +186,20 @@ The `sumologicexporter` sends data to Sumo in batches. If the batches are small,
...
 
+#### Does Sumo OpenTelemetry support HTTP checks?
+
+Yes, our OpenTelemetry collector supports HTTP checks, allowing you to ping public HTTPs URLs and monitor up/down status.
+
+How it works - the OpenTelemetry receiver produces metrics that can be ingested and then monitored to track non-200 responses, latency, and other errors.
+<!--
+1. To come...
+1. ...
+-->
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/faq.md:199]: @ajaiswals to provide (https://sumologic.atlassian.net/browse/SUMO-221966)

**Comment by kimsauce** [comment]
@ajaiswals @astencel-sumo , please advise on the steps ([Jira for reference](https://sumologic.atlassian.net/browse/SUMO-221966)).

File: docs/send-data/opentelemetry-collector/faq.md [hunk 5]
```diff
@@ -186,7 +186,36 @@ The `sumologicexporter` sends data to Sumo in batches. If the batches are small,
 
 If you are using the **Upstream OpenTelemetry collector** (instead of the **Sumo Logic OpenTelemetry collector**) and have a custom `config.yaml` file to collect data, then you might see this error. Sumo Logic expects specific tags on the data, which the Sumo Logic Distribution for OpenTelemetry Collector creates by default.
 
+## HTTP checks
 
+#### Does Sumo OpenTelemetry support HTTP checks?
+
+Yes, our OpenTelemetry Collector supports HTTP checks, allowing you to ping public HTTPs URLs and monitor up/down status.
+
+Here's how it works: the OpenTelemetry [HTTP Check receiver][httpcheck_receiver_docs] produces metrics that can be ingested and then monitored to track non-200 responses, latency, and other errors.
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/faq.md:195]:
```suggestion
Here's how it works: the OpenTelemetry [HTTP Check receiver][httpcheck_receiver_docs] produces metrics that can be ingested and then monitored to track non-200 responses, latency, and other errors. Below is a config sample.
```

**Pull request merged**